GlobalStyles/GlobalStyles.js

```diff
@@ -5,6 +5,9 @@ const GlobalStyles = createGlobalStyle`
     body {
         padding: 0;
         margin: 0;
+        position: relative;
```

Comment:
do you really need `position: relative`?

Reply:
just checked, you're right. I struggled with my footer, so I came up with this, but seems like it wasn't the solution :D I'll remove it.